client/app/lib/util/isFeatureEnabled.coffee

```diff
@@ -1,14 +1,21 @@
 globals   = require 'globals'
 checkFlag = require 'app/util/checkFlag'
 
+additionalChecks =
+  'gitlab' : ->
+    # for now this feature is only enabled for the team defined in config
+    globals.config.entryPoint.slug is globals.config.gitlab.team
+
 module.exports = (feature) ->
 
   isDisabled = globals.config.disabledFeatures[feature]
+  additionalCheckPassed = additionalChecks[feature]?() ? true
```

Comment:
I kept it singular on purpose since it's supporting only one additional check function at the moment, once I add multiple checks support I was planning to update it like that. But we can do it before hand :)